src/scalar_field.nr

```diff
@@ -340,4 +347,78 @@ mod tests {
         let scalar_field2 = scalar_field.into();
         assert(val as Field == scalar_field2);
     }
+    #[test]
+    fn test_get_modulus_slices() {
+        let modulus_slices: [u8; 64] = unsafe { get_modulus_slices::<64>() };
```

Comment:
These lines are throwing "Unnecessary unsafe block" warnings. Ideally this library should not be throwing warnings but we should avoid adding more.

Does your editor show these with yellow squiggly lines?